pgmpy/tests/test_models/test_DynamicBayesianNetwork.py

```diff
@@ -144,8 +144,10 @@ def test_get_interface_nodes(self):
         self.assertListEqual(
             sorted(self.network.get_interface_nodes()), [("D", 0), ("I", 0)]
         )
-        self.assertRaises(ValueError, self.network.get_interface_nodes, -1)
```

Comment:
These 2 tests were to just test if the argument checking part throws an error when an invalid time_slice is given. But might be failing right now because the argument check was removed.